src/libgdiplus.vcxproj

```diff
@@ -91,6 +91,7 @@
       <PreprocessorDefinitions>HAVE_LIBJPEG;HAVE_LIBTIFF;HAVE_LIBPNG;_WINDLL;WIN32;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <!-- FIXME: To align with the GDI+ calling convention, this should be StdCall. Only relevant on x86 -->
       <CallingConvention>Cdecl</CallingConvention>
+      <RuntimeLibrary Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">MultiThreadedDLL</RuntimeLibrary>
```

Comment:
I'm not an expert in this area, what I recall from my tests at the time is that you did need to make a change for Win32 but not for x64.

So I guess this could be replaced with `Condition="'$(Platform)'=='Win32'"`?